src/controllers/add_message.js

```diff
@@ -0,0 +1,16 @@
+const { postMessage } = require('../model/queries/');
+
+exports.get = (req, res) => { res.render('add_message'); };
+
+exports.post = (req, res, next) => {
+  // console.log(req.body);
+  const {
+    userId,
+    body,
+  } = req.body;
+  if (!body) res.render('add_message', { error: 'message' });
+  try {
```

Comment:
this is probably very sensible - but i think we should maybe try to figure out what the problem was with the use of promises throwing the 500 error...